src/radical/pilot/agent/staging_input/default.py

```diff
@@ -103,7 +104,7 @@ def _handle_unit(self, unit, actionables):
         # NOTE: see documentation of cu['sandbox'] semantics in the ComputeUnit
         #       class definition.
         sandbox = unit['unit_sandbox']
-
+        
```

Comment:
why?  :-)

Reply:
I think I had something there and I deleted it.

Reply:
done